### PR TITLE
added basic ssh host with existing key example

### PR DESCRIPTION
**Pull request by mfhughes**
super basic single host ssh functionality example done for/with alexander.

### CODE REVIEW

File: examples/ssh-host-agave-validator-python/__main__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""An SSH Python Pulumi program"""
+
+import pulumi
+import pulumi_tls as tls
+import pulumi_svmkit as svmkit
+
+ssh_key = """-----BEGIN OPENSSH PRIVATE KEY-----
+-----END OPENSSH PRIVATE KEY-----"""
```
Comment (kespinola) [inline examples/ssh-host-agave-validator-python/__main__.py:8]: Can you look at moving this to the Pulumi config and add a simple readme that describes encrypting the ssh key with pulumi?

File: examples/ssh-host-agave-validator-python/__main__.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "dynamicPortRange": "8002-8020",
+        "gossipPort": 8001,
+        "rpcBindAddress": "0.0.0.0",
+        "walRecoveryMode": "skip_any_corrupted_record",
+        "limitLedgerSize": 50000000,
+        "blockProductionMethod": "central-scheduler",
+        "fullSnapshotIntervalSlots": 1000,
+        "noWaitForVoteToStartLeader": True,
+    },
+#    opts=pulumi.ResourceOptions(depends_on=[instance])
```
Comment (kespinola) [inline examples/ssh-host-agave-validator-python/__main__.py:53]:
```suggestion
```

**Review by kespinola — CHANGES_REQUESTED** [review]
A few notes but excited to have you as our first outside contributor!

**Review by alexanderguy — APPROVED** [review]
LGTM; thank you for the contribution.

**Pull request merged**